guides/doc-Provisioning_Virtual/master.adoc

```diff
@@ -0,0 +1,51 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: provisioning-virtual
+:provisioning-virtual:
+
+= {ProvisioningVirtualDocTitle}
+
+// This guide is not ready for stable releases
```

Comment:
question: Do you have reasons to believe that the guide would not work for unstable and/or foreman-el/foreman-deb?

Reply:
It probably should work for all builds, since images are not part of Katello. However, right now I'm not sure it works at all and I would prefer to revisit this question later :))

Reply:
I vote for making this guide "working" for all stable releases. I'll keep it in mind during the review